core/src/main/scala/com/pingcap/tispark/utils/TwoPhaseCommitHepler.scala

```diff
@@ -86,16 +86,7 @@ case class TwoPhaseCommitHepler(startTs: Long, options: TiDBOptions) extends Aut
 
     secondaryKeysRDD.foreachPartition { partition =>
       val ti2PCClientOnExecutor =
-        new TwoPhaseCommitter(
-          tiConf,
-          startTs,
-          lockTTLSeconds * 1000,
-          options.txnPrewriteBatchSize,
```

Comment:
The following options can't work anymore?
It's a user-sensitive change

Reply:
> The following options can't work anymore? It's a user-sensitive change

agree. I will explain in the design document.